oauthlib/oauth2/rfc6749/parameters.py

```diff
@@ -133,14 +133,14 @@ def prepare_token_revocation_request(url, token, token_type_hint="access_token",
     using the "application/x-www-form-urlencoded" format in the HTTP request
     entity-body:
 
-    token   REQUIRED.  The token that the client wants to get revoked.
+    :param token:   REQUIRED.  The token that the client wants to get revoked.
 
-    token_type_hint  OPTIONAL.  A hint about the type of the token submitted
-    for revocation.  Clients MAY pass this parameter in order to help the
-    authorization server to optimize the token lookup.  If the server is unable
-    to locate the token using the given hint, it MUST extend its search across
-    all of its supported token types.  An authorization server MAY ignore this
-    parameter, particularly if it is able to detect the token type
+    :param token_type_hint:  OPTIONAL.  A hint about the type of the token
+    submitted for revocation.  Clients MAY pass this parameter in order to help
+    the authorization server to optimize the token lookup.  If the server is
+    unable to locate the token using the given hint, it MUST extend its search
+    across all of its supported token types.  An authorization server MAY ignore
+    this parameter, particularly if it is able to detect the token type
     automatically.  This specification defines two such values:
```

Comment:
Shouldn't we add `callback` and `body` as well ?

Reply:
yep. I somehow missed those.